src/host/stylesheets/iframe.less

```diff
@@ -77,13 +77,12 @@
         }
     }
 
-    .widgetOpened {
-        body,
-        html {
+    html.widgetOpened {
```

Comment:
The actual fix for the jumpy input is here... scope wasn't applied correctly on the page.